extensions/interactions/ItemSelectionInput/ItemSelectionInput.html

```diff
@@ -1,13 +1,55 @@
+<style>
+  .item-selection-form-container {
```

Comment:
Please prefix all class names in this file with ".item-selection-input-" (I know it's long, but we want to differentiate these classes from e.g. the bootstrap ones.)